app/controllers/serviceLearning/routes.py

```diff
@@ -79,10 +83,12 @@ def slcEditProposal(courseID):
                                         courseInstructor = courseInstructor,
                                         isAllSectionsServiceLearning = isAllSectionsServiceLearning,
                                         isPermanentlyDesignated = isPermanentlyDesignated,
+                                        hasSlcComponent = hasSlcComponent,
```

Comment:
We are already passing `course` to the template. Can't it just use `course.hasSlcComponent` instead of creating a new variable? I guess that's true of the existing two variables that are similarly unnecessary.